app/models/tweets_tag.rb

```diff
@@ -6,7 +6,7 @@ class TweetsTag
   with_options presence: true do
     validates :message
     validates :name
-    validates :content
+    validates :content_required
```

Comment:
`validates` メソッドはクラスのattributeに対して用いるメソッドです。
https://api.rubyonrails.org/classes/ActiveModel/Validations/ClassMethods.html#method-i-validates
`content` attributeに対して適用したいのであればこうなりますね
```suggestion
    validates :content
```

Reply:
自身で定義したメソッド(今回の例だと `content_required` メソッド)をバリデーションとして使いたい場合は `validate` メソッドを使いましょう。
https://api.rubyonrails.org/classes/ActiveModel/Validations/ClassMethods.html#method-i-validate
https://railsguides.jp/active_record_validations.html#カスタムメソッド

`with_options` のブロック内に書く必要はないのでこうなりますね

```rb
with_options presence: true do
  validates :message
  validates :name
end

validate :content_required
```